docs/ocean/sapi_access_basic.rst

```diff
@@ -110,8 +110,7 @@ environment from either Ocean's default values or your configuration.
 *   **Solver** (default: feature-based selection)
 
     A |dwave_short| resource to be used to solve your submitted problems; for
-    example, a :term:`hybrid` :term:`solver` or an :term:`Advantage` quantum
-    computer.
+    example, a :term:`hybrid` or QPU :term:`solver`.
```

Comment:
```suggestion
    example, a :term:`hybrid` :term:`solver` or quantum computer.
```

This is a page for beginners and we don't explain QPU on it. It's true that they tooltip explains but it seems simpler to just spell out quantum computer here